src/transformers/feature_extraction_common_utils.py

```diff
@@ -13,48 +13,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
- Feature extraction common class for python feature extractors.
+ Feature extraction saving/loading class for common feature extractors.
 """
+
 import copy
 import json
 import os
 from collections import UserDict
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Union
 
 import numpy as np
 
 from .file_utils import (
     FEATURE_EXTRACTOR_NAME,
-    PaddingStrategy,
     TensorType,
     _is_jax,
     _is_numpy,
-    _is_tensorflow,
-    _is_torch,
     _is_torch_device,
     cached_path,
     hf_bucket_url,
     is_flax_available,
     is_remote_url,
     is_tf_available,
     is_torch_available,
-    to_py_obj,
     torch_required,
 )
 from .utils import logging
 
 
-logger = logging.get_logger(__name__)
-
-
 if TYPE_CHECKING:
     if is_torch_available():
         import torch
 
 
+logger = logging.get_logger(__name__)
+
+PreTrainedFeatureExtractor = Union["PreTrainedSequenceFeatureExtractor"]  # noqa: F821
```

Comment:
I guess this type will be filled with the feature extractors added down the line?